### ISSUE TITLE
fix docker container

### ISSUE THREAD

**Issue by nllong** [SEED-platform/seed#366]
A few items broke after the upgrade and repo reorg.

- [x] COPY ./requirements.txt /seed/requirements.txt to COPY ./requirements/*.txt /seed/requirements/. Check which file to COPY.
- [x] export DJANGO_SETTINGS_MODULE=BE.settings.docker to config.settings.docker
- [x] Add .dockerignore to build faster
- [x] Add docker_compose file
- [x] Update the installation instructions in the wiki to include a step about local_untracked.py; currently there is local_untracked.py.dist, which needs to be copied over and configured before the “Setup the initial demo user” step (database needs to be configured)
- [x] Under “Start the Web and Celery services”, -name should be –name:
- [x]  Wsgi.py has moved, need to update seed/bin/start_uwsgi_docker.sh to match:

```
docker run -d --name seed-uwsgi -v $HOME/seed_data:/seed/collected_static --link seed-redis:redis --link seed-postgres:postgres -p 8000:8000 seed-platform /seed/bin/start_uwsgi_docker.sh
docker run -d --name seed-celery -v $HOME/seed_data:/seed/collected_static --link seed-redis:redis --link seed-postgres:postgres seed-platform /seed/bin/start_celery_docker.sh
NEW_RELIC_CONFIG_FILE=newrelic.ini newrelic-admin run-program /usr/local/bin/uwsgi --logto /seed/logs/seed.log --logto2 /seed/logs/seedERR.log --http 0.0.0.0:8000<http://0.0.0.0:8000> --module config.wsgi --max-requests 5000 --pidfile /var/run/uwsgi.pid --cheaper-initial 1 -p $WORKERS --single-interpreter --enable-threads
```

**Comment by nllong** [comment]
PR #695

**Comment by nllong** [comment]
@RDmitchell this was fixed with PR #695 and I am closing the issue.

**Comment by RDmitchell** [comment]
great !
